static/app/views/alerts/rules/metric/triggers/chart/index.tsx

```diff
@@ -222,6 +222,9 @@ class TriggersChart extends PureComponent<Props, State> {
     }
   }
 
+  // Create new API Client so that historical requests aren't automatically deduplicated
+  historicalAPI = new Client();
```

Comment:
New API Client just for historical requests

Reply:
Are you thinking that when we start with ACI that these API requests would go through tanstack/query?

Reply:
Yep! This will all go through a refactor to move to functional components and the new API hooks.